styles/overrides/_popover.scss

```diff
@@ -2,9 +2,11 @@
   font-family: $typo-stack-base;
   box-shadow: $shadow-base;
   border-color: $neutral-light;
-  @include font-smoothing;
   animation: dropdownAnimation 750ms;
   animation-fill-mode: forwards;
+  @media (-webkit-min-device-pixel-ratio: 1.25), (min-resolution: 120dpi) {
```

Comment:
This is added via the global styling. I think we just need to remove `@include font-smoothing;` from the existing code 👍